src/deepsparse/engine.py

```diff
@@ -572,9 +572,10 @@ def __init__(
         self,
         num_cores: int = None,
         num_streams: int = None,
+        scheduler: Scheduler = Scheduler.elastic,
```

Comment:
mentioned offline - we should only be supporting elastic scheduler with `Context`

Reply:
we should just remove this, no?

Reply:
Oh yeah - will revert now